BrowserKit/Tests/SiteImageViewTests/ImageProcessing/FaviconFetcherTests.swift

```diff
@@ -49,42 +49,24 @@ final class FaviconFetcherTests: XCTestCase {
             XCTFail("Should have succeeded with image")
         }
     }
-
-    func testTimeout_completesWithoutImageOrError() async {
```

Comment:
This test didn't make sense anymore because we're using Kingfisher's timeout functionality.